hack/generate/samples/internal/go/v3/memcached_with_webhooks.go

```diff
@@ -100,6 +101,11 @@ func (mh *Memcached) Run() {
 	mh.uncommentDefaultKustomization()
 	mh.uncommentManifestsKustomization()
 
+	cmd := exec.Command("go", "mod", "tidy")
+	cmd.Dir = mh.ctx.Dir
+	_, err = mh.ctx.Run(cmd)
+	pkg.CheckError("Running go mod tidy", err)
+
```

Comment:
should not be done in KB ? 
Should we upgrade SDK before upgrade KB?

Reply:
Kubebuilder has been upgraded to go 1.16.